teams/libs.toml

```diff
@@ -3,18 +3,20 @@ name = "libs"
 [people]
 leads = ["alexcrichton"]
 members = [
-    "alexcrichton",
-    "sfackler",
+    "Amanieu",
     "BurntSushi",
     "Kimundi",
+    "KodrAus",
     "SimonSapin",
+    "alexcrichton",
     "dtolnay",
-    "aturon",
-    "KodrAus",
+    "sfackler",
     "withoutboats",
-    "Amanieu",
 ]
 
+[github]
+orgs = ["rust-lang"]
```

Comment:
`@rust-lang-nursery/libs` exists, want to sync with it as well?

Reply:
Good point!